maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```
@@ -434,7 +434,7 @@ class IOSDriver(
     override fun inputText(text: String) {
         iosDevice.input(
             text = text,
-        ).expect {}
+        )
```

Could you add a comment string mentioning the silent fail?